Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/UsernamePassword.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class UsernamePassword : HeaderBasedAuthorization
+    {
+        public string username;
+        public string password => key;
+        public override string headerPrefix { get; protected set; } = "Basic ";
```

Comment:
Zou niet public moeten zijn - dit is een intern gegeven

Reply:
Na het lezen van de rest van de code, als we een public property of method introduceren "HeaderValue" in de HeaderBasedAuthorization dan hoeven we minder in de aanroeper te doen wat eigenlijk hierin hoort - en kunnen we die code ook versimpelen